packages/gdl-frontend/components/FeaturedContentCarousel/Carousel.js

```diff
@@ -0,0 +1,242 @@
+                  </CardContent>
+                </HeroCardTablet>
+              </Banner>
+
+              <HeroCardMobile>
+                <CardContent>
+                  {cardContent(content)}
+                  <Hidden only="mobile">
+                    <div style={{ paddingTop: '8px' }}>
+                      <Pagination
+                        dots={featuredContent.length}
+                        index={index}
+                        onChangeIndex={index => this.setState({ index })}
+                      />
+                    </div>
+                  </Hidden>{' '}
+                </CardContent>
+              </HeroCardMobile>
+            </div>
+          ))}
+        </AutoPlaySwipeableViews>
+        <Hidden only="desktop">
+          {(index !== 0 && (
+            <div
+              css={arrowLeftContainer}
+              aria-label="Previous"
+              onClick={this.handlePrevIndex}
+            >
+              {' '}
+              <KeyboardArrowLeft style={{ color: 'white' }} />
+            </div>
+          )) ||
+            (index === 0 && (
+              <div
+                css={arrowLeftContainer}
+                aria-label="Previous"
+                onClick={this.goToLastPage}
+              >
+                {' '}
+                <KeyboardArrowLeft style={{ color: 'white' }} />
+              </div>
+            ))}
+          {(index !== featuredContent.length - 1 && (
+            <div
+              css={arrowRightContainer}
+              aria-label="Next"
+              onClick={this.handleNextIndex}
+            >
+              {' '}
+              <KeyboardArrowRight style={{ color: 'white' }} />
+            </div>
+          )) ||
+            (index === featuredContent.length - 1 && (
+              <div
+                css={arrowRightContainer}
+                aria-label="Next"
+                onClick={this.goToFirstPage}
+              >
+                {' '}
+                <KeyboardArrowRight style={{ color: 'white' }} />
+              </div>
+            ))}
+          <div css={dotsContainer}>
+            <Pagination
+              dots={featuredContent.length}
+              index={index}
+              onChangeIndex={index => this.setState({ index })}
+            />
+          </div>
+        </Hidden>
+        <Hidden only="tablet">
+          <div css={dotsContainer}>
+            <Pagination
+              dots={featuredContent.length}
+              index={index}
+              onChangeIndex={index => this.setState({ index })}
+            />
+          </div>
+        </Hidden>
+      </div>
+    );
+  }
+}
+
+export default Carousel;
+
+const arrowRightContainer = css`
+  position: absolute;
+  width: 9.9%;
+  height: 100%;
+  top: 0;
+  right: 0;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  flex-direction: row-reverse;
+  &:hover {
+    transition: all 0.2s ease-in;
+    background: rgba(0, 0, 0, 0.5);
```

Comment:
Foreslår en gradient farge som blir lit "finere" ```    background-image: linear-gradient(
      90deg,
      rgba(207, 207, 207, 0.1),
      rgba(0, 0, 0, 0.5)
    );```
Bare å leke deg med fargen for å finne noe enda bedre.
Merk at `90deg` blir `-90deg` for `arrowLeftContainer`